courses/machine_learning/deepdive2/text_classification/labs/reusable_embeddings.ipynb

```diff
@@ -16,7 +16,7 @@
     "## Introduction\n",
```

Comment:
<p>Can we remove the TODO's here and include the full code, since we are just bringing this "Create a Dataset from BigQuery" from the previous lab to make this notebook independent. Just use the solution notebook code. </p><p> </p><p>This will keep the learning objectives to "Reusable Embeddings" and keep the duration of the lab shorter. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/863/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I have removed the TODO items in the commit https://github.com/GoogleCloudPlatform/training-data-analyst/pull/863/commits/75686e169e7ba64ac685d80d3e83eb6d4fe11c2d